core/templates/pages/contributor-dashboard-page/modal-templates/translation-suggestion-review-modal.component.spec.ts

```diff
@@ -1095,139 +2124,215 @@ describe('Translation Suggestion Review Modal Component', function () {
     }));
   });
 
-  describe(
```

Comment:
Why are we removing these?

Reply:
I am not removing any existing tests here, I have introduced two new describe blocks and also renamed two of the existing ones.

The new ones have suffix as `when flag CdAllowUndoingTranslationReview is enabled` and two existing ones which I renamed have suffix `flag CdAllowUndoingTranslationReview is disabled`.

The reason, I did this was to make sure that clean up of the feature flag remains simple once everything is tested in feature testing.